content/2022-02-01-github-gateway-goes-ga.md

```diff
@@ -0,0 +1,51 @@
+---
+title: "The Brigade GitHub Gateway Goes GA"
+slug: "2022-02-01-github-gateway-goes-ga"
+description: "The new Brigade GitHub Gateway is now generally available"
+date: "2022-02-01"
+author: "Kent Rancourt"
+summary: The Brigade GitHub Gateway is now generally available.
+---
+
+Big news, Brigadiers!
+
+Since Brigade v2's release two months ago, the team's shifted its attention to
+making our various v2-compatible gateways production-ready, and today we're
+proud to announce that the first of those -- one we perceive as extraordinarily
+useful -- has reached general availability. Today we'd like to introduce you
+to the all new
+[Brigade GitHub Gateway](https://github.com/brigadecore/brigade-github-gateway).
+
+As you might infer from its name, the Brigade GitHub Gateway transforms events
+(webhooks) from GitHub into _Brigade_ events and emits them into Brigade's event
+bus. Although the gateway works with any and all events emitted by a
+[GitHub App](https://docs.github.com/en/developers/apps), its most common use
+case is as a trigger for CI/CD pipelines that execute within Brigade -- and, in
+fact, this is exactly how Brigade's own CI/CD pipelines have been implemented.
+
+Putting aside its obvious utility, what the maintainers are most excited about
```

Comment:
I'm always going to flag words like "obvious" because seeing that word makes me go back and read the previous paragraph a few times to see if I missed something. Do you need an adjective here at all? And if you do, can you use one like "practical" or "inherent"?

Reply:
Fixed in 9ba1e39bcf2eb6a5e501fe34c1f4d3559b8c69d6